gcexport.py

```diff
@@ -956,21 +1028,21 @@ def main(argv):
                 else:
                     start_time_seconds = None
 
-                extract['device'] = extract_device(device_dict, details, start_time_seconds, args, http_req, write_to_file)
+                extract['device'] = extract_device(device_dict, details, start_time_seconds, args, http_req_as_string, write_to_file)
 
                 # try to get the JSON with all the samples (not all activities have it...),
                 # but only if it's really needed for the CSV output
                 extract['samples'] = None
                 if csv_filter.is_column_active('sampleCount'):
                     try:
                         # TODO implement retries here, I have observed temporary failures
-                        activity_measurements = http_req(URL_GC_ACTIVITY + str(actvty['activityId']) + "/details")
+                        activity_measurements = http_req_as_string(URL_GC_ACTIVITY + str(actvty['activityId']) + "/details")
```

Comment:
```suggestion
                        activity_measurements = http_req_as_string(f'{URL_GC_ACTIVITY}{actvty['activityId']}/details')
```

Reply:
As for Python2 my suggestion should be dropped and keep the original change from the PR